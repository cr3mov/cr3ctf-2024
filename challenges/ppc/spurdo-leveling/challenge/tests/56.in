82503174867583272 928044503597795238
40 71857
100 79939
