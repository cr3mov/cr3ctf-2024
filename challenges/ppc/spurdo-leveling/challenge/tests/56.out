22280403919159 5570100979779 5570100979776
